controller_manager/test/test_controller_manager_srvs.cpp

```diff
@@ -603,22 +604,6 @@ TEST_F(TestControllerManagerSrvs, list_sorted_chained_controllers)
   result = call_service_and_wait(*client, request, srv_executor);
   ASSERT_EQ(6u, result->controller.size());
 
-  // activate controllers
-  cm_->switch_controller(
-    {TEST_CHAINED_CONTROLLER_1}, {},
-    controller_manager_msgs::srv::SwitchController::Request::STRICT, true, rclcpp::Duration(0, 0));
-  cm_->switch_controller(
-    {TEST_CHAINED_CONTROLLER_3, TEST_CHAINED_CONTROLLER_5, TEST_CHAINED_CONTROLLER_2,
-     TEST_CHAINED_CONTROLLER_4},
-    {}, controller_manager_msgs::srv::SwitchController::Request::STRICT, true,
-    rclcpp::Duration(0, 0));
-  cm_->switch_controller(
-    {test_controller::TEST_CONTROLLER_NAME}, {},
-    controller_manager_msgs::srv::SwitchController::Request::STRICT, true, rclcpp::Duration(0, 0));
-  // get controller list after activate
-  result = call_service_and_wait(*client, request, srv_executor);
-
-  ASSERT_EQ(6u, result->controller.size());
```

Comment:
It seams to me that we are loosing those tests here. I would expect to have the same test just before controllers are configured. We have to make sure that this code is well tested, so we need different combinations of unordered controllers that are getting loaded randomly, but are sorted properly after configuration.

Reply:
OK, I understand. Thanks for the explanation!